libs/core/state/src/errors/transform-error-to-state-error.ts

```diff
@@ -0,0 +1,10 @@
+import { DaffError } from '@daffodil/core';
+
+import { DaffStateError } from './state-error.interface';
+
+/**
+ * Transforms an error instance to a state error object.
+ */
+export function daffTransformErrorToStateError(error: DaffError): DaffStateError {
+  return error;
```

Comment:
This is safe, for now, but wouldn't be in the future, please explicitly meet the type cast.

Reply:
done